nvm.sh

```diff
@@ -1010,14 +1045,14 @@ nvm_install_node_source() {
   if (
     [ -n "$tarball" ] && \
     command mkdir -p "$tmpdir" && \
-    nvm_download -L --progress-bar $tarball -o "$tmptarball" && \
+    nvm_download -L "$PROGRESS_BAR" $tarball -o "$tmptarball" && \
     nvm_checksum "$tmptarball" $sum && \
     command tar -xzf "$tmptarball" -C "$tmpdir" && \
     cd "$tmpdir/node-$VERSION" && \
-    ./configure --prefix="$VERSION_PATH" $ADDITIONAL_PARAMETERS && \
-    $make $MAKE_CXX && \
+    eval ./configure --prefix="$VERSION_PATH" $ADDITIONAL_PARAMETERS ${REDIRECT_OUTPUT} && \
```

Comment:
I'm very concerned about the addition of `eval`. Will this work in POSIX? `nvm` must work in all of `sh`, `dash`, `zsh`, `bash`, and `ksh`.